hibernate-core/src/main/java/org/hibernate/SharedSessionContract.java

```diff
@@ -68,6 +68,17 @@ public interface SharedSessionContract extends QueryProducer, AutoCloseable, Ser
 	 * Begin a unit of work and return the associated {@link Transaction} object.
 	 * If a new underlying transaction is required, begin the transaction. Otherwise,
 	 * continue the new work in the context of the existing underlying transaction.
+	 * <p>
+	 * The JPA-standard way to begin a new transaction is by calling
```

Comment:
Since you are calling out JPA-specific expectations here, might also be worth noting here that JPA disallows this in JTA environments and we also honor that with strict compliance.

Reply:
Updated to address this.